pkg/api/api.go

```diff
@@ -320,7 +321,7 @@ func (s *Service) getOrCreateTag(tagUid string) (*tags.Tag, bool, error) {
 func (s *Service) getTag(tagUid string) (*tags.Tag, error) {
 	uid, err := strconv.Atoi(tagUid)
 	if err != nil {
-		return nil, fmt.Errorf("cannot parse taguid: %w", err)
+		return nil, fmt.Errorf("err %v: %w", errCannotParse, err)
```

Comment:
The `%v` verb just converts the error to a string by calling the `Error() string` method. As a result, error checking against this error would not be possible (as is done in the case of calling the `getTag` function.